great_tables/_tbl_data.py

```diff
@@ -313,38 +329,64 @@ def _(data: PlDataFrame, expr: Union[list[str], _selector_proxy_], strict: bool
     from operator import or_
```

Comment:
Are these legacy imports?

Reply:
I'm referring to the lines `from functools import reduce` and `from operator import or_`.